test_generated_models.sh

```diff
@@ -1,44 +1,55 @@
 #!/bin/bash
-cur_dir=`pwd`
-mkdir -p ${cur_dir}/models/
-mkdir -p ${cur_dir}/temp/
-mkdir -p ${cur_dir}/shapes/
-mkdir -p ${cur_dir}/submodules/
+CORES=`lscpu | grep Core | awk '{print $4}'`
+SOCKETS=`lscpu | grep Socket | awk '{print $2}'`
+TOTAL_CORES=`expr $CORES \* $SOCKETS - 1`
+benchmark_dir="/home/pnp/yudongsi/benchmark"
```

Comment:
Please avoid using specific paths in the scripts. Suggest replacing it with an external environment variable.

Reply:
Please do not remove those three lines for computing TOTAL_CORES, which will be used for the performance test in numactl. Not related to BS

Reply:
ok